packages/material-ui/src/Modal/ModalManager.js

```diff
@@ -22,8 +22,12 @@ export function ariaHidden(node, show) {
   }
 }
 
+function getPaddingRightRaw(node) {
+  return parseInt(window.getComputedStyle(node)['padding-right'], 10) || '';
+}
+
 function getPaddingRight(node) {
```

Comment:
The usage of this method (in the preivous logic) doesn't seem to be consitent. Why would we use it for the .mui-fixed class names and not for the body?